apps/designer/app/designer/features/sidebar-left/panels/asset-manager/asset-tooltip.tsx

```diff
@@ -46,8 +46,17 @@ export const AssetTooltip = ({
           top: "$1",
           right: "$1",
           cursor: "pointer",
-          color: "$hiContrast",
           transition: "opacity 100ms ease",
+
+          "svg path": {
+            fill: "$loContrast",
+          },
+
+          "&:hover": {
+            "svg path, svg circle": {
+              stroke: "$hiContrast",
```

Comment:
you coulnd't just set the current color?

Reply:
That would allow you to not write those svg selectors

Reply:
same for fill

Reply:
I did current color but still had to use the `svg` because of CSS specificity

Reply:
why? who is also defining that color? specificity means there is a conflict

Reply:
https://github.com/webstudio-is/webstudio-designer/blob/ad4286782ccb726e11ab1e426f53975b2dd15db7/apps/designer/app/designer/features/sidebar-left/panels/asset-manager/asset-thumbnail.tsx#L71

Reply:
I would fix both and use props instead of cascading selectors. Prop that is semantically saying what it is changing.

Reply:
I used JS for hover and passed props down